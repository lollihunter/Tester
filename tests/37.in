100 100 100
1 1
86 37
14 91
57 95
97 39
31 70
63 16
53 49
94 99
81 2
74 7
17 24
19 48
57 51
48 37
80 42
80 59
97 37
39 3
92 42
75 99
72 32
90 1
87 15
98 76
100 38
8 97
74 90
61 81
32 76
100 53
15 69
76 54
31 7
61 39
8 1
96 11
72 18
6 77
16 63
81 80
30 24
91 54
42 1
33 49
37 82
17 94
23 88
65 62
78 99
43 86
64 53
78 50
41 61
34 38
77 58
76 37
37 15
18 20
22 68
70 76
60 99
86 64
18 32
88 13
26 74
27 63
82 56
26 13
23 49
49 86
78 69
99 16
81 32
84 86
40 76
19 1
37 66
84 83
19 10
35 33
46 94
13 15
88 18
80 78
63 85
8 89
52 63
13 49
65 91
31 76
91 92
92 33
43 21
55 7
21 1
100 40
52 45
73 100
100 100
