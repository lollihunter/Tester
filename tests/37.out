29
1 36 80 69 34 58 42 81 55 45 54 78 6 30 46 51 71 4 90 86 61 72 85 41 79 75 92 9 100 
